modules/ddl-and-loading/pages/functions/token/gsql_ts_to_epoch_legacy.adoc

```diff
@@ -1 +1,49 @@
 = gsql_ts_to_epoch_seconds_legacy()
+
+Converts a timestamp in canonical string format to Unix epoch time (seconds since `Jan 1, 1970`).
+If the timestamp is invalid or earlier than `1970-01-01 00:00:00`, the function returns a default value (0 by default), ensuring backward compatibility and preventing exceptions.
+
+We also have another timestamp function, xref:ddl-and-loading:functions/token/gsql_ts_to_epoch.adoc[gsql_ts_to_epoch_seconds()], which converts a timestamp to Unix epoch time (seconds since Jan 1st, 1970) and returns an unsigned integer (`UINT`). For broader date range support, consider xref:ddl-and-loading:functions/token/gsql_ts_to_epoch_signed.adoc[gsql_ts_to_epoch_seconds_signed()].
```

Comment:
```suggestion
We also have another timestamp function, xref:ddl-and-loading:functions/token/gsql_ts_to_epoch.adoc[gsql_ts_to_epoch_seconds()], which converts a timestamp to Unix epoch time (seconds since Jan 1st, 1970) and throws an exception instead of returning a user-specified value, if the input is not valid. For broader date range support, consider xref:ddl-and-loading:functions/token/gsql_ts_to_epoch_signed.adoc[gsql_ts_to_epoch_seconds_signed()].
```